### ISSUE TITLE
dont discard logs

### ISSUE THREAD

**Issue by colttt** [Graylog2/graylog2-server#11124]
## What?

It happens that logs are discarded because the extractor/converter doesn't match. It would be usefull if graylog doesn't discard the logs, instead just skip the extrator thing and add leave the message as is and add an field like 'graylog_note' which can contain the error message (just an example)
> ERROR [CsvConverter] Different number of columns in CSV data (5) and configured field names (6).
and then we can adjust the extractor and no message will be lost

## Why?

**no** message will be lost, which is very important

## Your Environment

* Graylog Version:
* Elasticsearch Version:
* MongoDB Version:
* Operating System:
* Browser version:

**Comment by colttt** [comment]
@bernd can you please change the label from feature to bug?

because graylog is a log management tool and a log management tool must never ever discard logs if so, its useless. it can always happen that are logs coming in that are not parseable.

**Comment by ESIC-DA** [comment]
Yeah, this would be super helpful to have....trying to debug an extractor right now and tearing my hair out found me this...

In the meantime anyone have any suggestions? I cant even find an example for the needle in the haystack messages that are causing this in order to weed them out via ...regexping harder I guess.

Also the fact that any message gets dropped entirely isnt exactly filling me with warm fuzzies (so far as I can tell, i don't see any non-extracted messages during the timespan that CsvExtractor is complaining in the graylog log.

**Comment by ESIC-DA** [comment]
Also: I agree that dropping messages from graylog, no matter the reason, should count as a *bug*, so far as I can tell, there is no way to tell it to keep but not parse the message...but if I'm wrong on that, let me know.

**Comment by colttt** [comment]
any news for that?

**Comment by colttt** [comment]
only a enterprise feature?! thats sad :-(  any plans to get this in the community edition?
in the docs its not clear that is just a enterprise feature

**Comment by boosty** [comment]
> any plans to get this in the community edition?

So far there are no plans to add this to Graylog Open.
